Tests/ApodiniTests/OpenAPITests/OpenAPIDocumentBuilderTests.swift

```diff
@@ -125,9 +125,9 @@ final class OpenAPIDocumentBuilderTests: ApodiniTests {
 
         let builtDocument = documentBuilder.build()
 
-        XCTAssertNoThrow(try builtDocument.output(.json))
-        print(try document.output(.json)!)
-        print(try builtDocument.output(.json)!)
+        XCTAssertNoThrow(try builtDocument.output(configuration: exporterConfiguration))
+        print(try document.output(configuration: exporterConfiguration)!)
+        print(try builtDocument.output(configuration: exporterConfiguration)!)
```

Comment:
Left over debug prints! Please also search the rest of the project...maybe I oversaw some...

Reply:
I actually didn't insert those prints (I think they got in via Metadata DSL or smt. like that), I just edited how the configuration is passed

Reply:
Ah, I see...that happens...please remove them anyways.